net/nettest/listener.go

```diff
@@ -61,7 +61,7 @@ func (l *Listener) Accept() (net.Conn, error) {
 // The provided Context must be non-nil. If the context expires before the
 // connection is complete, an error is returned. Once successfully connected
 // any expiration of the context will not affect the connection.
-func (l *Listener) Dial(ctx context.Context, network, addr string) (net.Conn, error) {
+func (l *Listener) Dial(ctx context.Context, network, addr string) (_ net.Conn, cerr error) {
```

Comment:
Can you change this `cerr` to just `err`? I don't know what `c` is supposed to mean and `err` is the idiomatic name.

Reply:
Done, thank you @bradfitz!